### PR TITLE
New package: Moonlight-4.1.0

### PR DESCRIPTION
**Pull request by TinfoilSubmarine**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

File: srcpkgs/Moonlight/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'Moonlight'
+pkgname=Moonlight
+version=3.1.4
+revision=1
+wrksrc="Moonlight-${version}"
```
Comment (abenson) [inline srcpkgs/Moonlight/template:5]: This is the default and can be removed.

**Comment by abenson** [comment]
As I understand this can only stream from a system with GeForce Experience (which is Windows-only, I believe)?

**Comment by TinfoilSubmarine** [comment]
That may be the case, that's how me/the person I'm working on this for use it currently.

File: srcpkgs/Moonlight/template (hunk 1 above)
Comment (TinfoilSubmarine) [reply srcpkgs/Moonlight/template]: fixed, thanks!

**Pull request closed without merging**